common/thanos/templates/alerts/_thanos-compactor.alerts.tpl

```diff
@@ -3,83 +3,109 @@
 groups:
 - name: thanos-compactor.alerts
   rules:
-    - alert: ThanosCompactHalted
-      expr: thanos_compact_halted{app="thanos-compactor", prometheus="{{ include "thanos.name" . }}"} == 1
+    - alert: ThanosCompactMultipleRunning
+      expr: sum by (thanos) (up{job=~".*thanos.*compact.*", thanos="{{ include "thanos.name" . }}"}) > 1
       for: 5m
       labels:
-        context: thanos
         service: {{ default "metrics" $root.Values.alerts.service }}
-        support_group: observability
-        severity: info
-        playbook: 'docs/support/playbook/prometheus/thanos_compaction.html'
-        meta: 'Thanos compaction for Prometheus `{{`{{ $labels.prometheus }}`}}` halted.'
+        support_group: {{ default "observability" $root.Values.alerts.support_group }}
+        severity: warning
+        meta: Multiple Thanos compactors running for `{{`{{ $labels.thanos }}`}}`.
       annotations:
-        description: 'Thanos compaction has failed to run and now is halted for Prometheus `{{`{{ $labels.prometheus }}`}}`. Long term storage queries will be slower.'
-        summary: Thanos compaction halted
+        description: |
+          No more than one Thanos Compact instance should be running
+          at once for `{{`{{ $labels.thanos }}`}}`.
+          Metrics in long term storage may be corrupted.
+        summary: Thanos Compact has multiple instances running.
 
-    - alert: ThanosCompactCompactionsFailed
-      expr: rate(prometheus_tsdb_compactions_failed_total{app="thanos-compactor", prometheus="{{ include "thanos.name" . }}"}[5m]) > 0
+    - alert: ThanosCompactHalted
+      expr: thanos_compact_halted{job=~".*thanos.*compact.*", thanos="{{ include "thanos.name" . }}"} == 1
+      for: 5m
       labels:
-        context: thanos
         service: {{ default "metrics" $root.Values.alerts.service }}
-        support_group: observability
+        support_group: {{ default "observability" $root.Values.alerts.support_group }}
         severity: info
         playbook: 'docs/support/playbook/prometheus/thanos_compaction.html'
```

Comment:
FWIW: the playbook does not exist

Reply:
Yeah, I know. Let's create the playbooks step by step afterwards when we need to respond to one of the new alerts. Monitoring the #cc-metrics channel after deployment.
```suggestion
```